Library/Homebrew/test/formula_pin_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "formula_pin"
+
+describe FormulaPin do
+  subject { described_class.new(formula) }
+  let(:formula) { formula_double_class.new }
```

Comment:
Couldn't we just use a `double`? This seems like a very complicated way to do a very simple thing.